src/axom/sidre/tests/CMakeLists.txt

```diff
@@ -104,17 +104,26 @@ endif()
 #------------------------------------------------------------------------------
 # Add MFEMSidreDataCollection tests
 #------------------------------------------------------------------------------
-if (AXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION)
+if (MFEM_FOUND AND AXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION)
+    set(_sidre_mfem_dc_depends axom gtest)
+    blt_list_append(TO _sidre_mfem_dc_depends ELEMENTS mpi IF ENABLE_MPI)
```

Comment:
mpi should have already been inherited by the axom target.  are we sure this needs to be added?